cogniac/app.py

```diff
@@ -168,9 +168,12 @@ def __setattr__(self, name, value):
 
         super(CogniacApplication, self).__setattr__(name, value)
 
-    def __repr__(self):
+    def __str__(self):
```

Comment:
does this class need @six.python_2_unicode_compatible too ?

Reply:
I did mean to include it, I don't think it strictly requires it because it originally only had a __repr__() without encoding the string for printing, but might as well include for potential unicode compatibility and consistency with the other classes?

Reply:
The app name can contain non-ascii characters, I think it should be similar to subject class.